examples/qtensor/qtensor.morpho

```diff
@@ -1,17 +1,26 @@
+/* Problem of a 2D nematic confined to a disk with parallel anchoring boundary condition, modeled by a Q-tensor free energy.
+Showcases: AreaIntegral, LineIntegral, MeshRefiner, POVRaytracer, Graphics
+This script showcases a few good aspects of Morpho:
```

Comment:
> This script showcases a few good aspects of Morpho

This makes it sound like morpho has a lot of bad aspects

Reply:
Maybe just replace this with: "This example demonstrates the following features of morpho:"